### DISCUSSION TITLE
Feature Request Review

### DISCUSSION THREAD

**Discussion by krakhead22** [jellyfin/jellyfin#11036] [Q&A]
I recently added a feature request (https://features.jellyfin.org/posts/2572/allow-movies-as-tv-specials) which I feel was incorrectly marked a duplicate by the moderators. The feature request they tagged is the opposite of mine and would involve a completely different solution. Is there a way to request a second opinion as once a feature is marked duplicate it appears to not be searchable and is thus essentially invisible.

**Comment by JPVenson** [comment]
Hey,
I relayed the request. In future, its much easier to hop into our chat to reach us for this kind of questions.
